framework/lib/components/file-picker/file-picker.tsx

```diff
@@ -205,7 +205,11 @@ export const FilePicker = ({
         onMouseEnter={ () => setEditable(true) }
         onMouseLeave={ () => setEditable(false) }
       >
-        <Box sx={ filePicker }>
+        <Box sx={ {
+          ...filePicker,
+          height: rest?.height ?? filePicker.height,
+        } }
+        >
```

Comment:
I think it would be nice if this worked for other properties as well and not just height. Perhaps we could merge the sx prop with the default filePicker styles like it is done in `drop-zone.tsx` and `theme/components/drag-and-drop/index.ts`